sample/dns-example.c

```diff
@@ -196,7 +196,16 @@ main(int c, char **v) {
 #endif
 
 	event_base = event_base_new();
+	if (event_base == NULL) {
+		fprintf(stderr, "Couldn't create new event_base\n");
+		return 1;
+	}
 	evdns_base = evdns_base_new(event_base, EVDNS_BASE_DISABLE_WHEN_INACTIVE);
+	if (evdns_base == NULL) {
```

Comment:
At least this code path leaks `event_base`

Reply:
OK, I'll fix it